pkgs/top-level/perl-packages.nix

```diff
@@ -13201,7 +13204,7 @@ let
       url = "mirror://cpan/authors/id/M/MA/MATTN/Module-Install-ReadmeMarkdownFromPod-0.04.tar.gz";
       sha256 = "300b2e244f83b9a54a95f8404c1cd3af0635b4fae974ca65390ee428ec668591";
     };
-    buildInputs = [ URI ];
```

Comment:
this may be wrong, it is one of the packages you recently added, and cpan2nix has no glue. it took deps from cpan metadata, which is often invalid